include/setup.h

```diff
@@ -129,7 +129,13 @@ class Value {
 class Property {
 public:
 	struct Changeable {
-		enum Value { Always, WhenIdle, OnlyAtStart, Deprecated };
+		enum Value {
+			Always,
+			WhenIdle,
+			OnlyAtStart,
+			Deprecated,
+			DeprecatedButAllowed
+		};
 	};
```

Comment:
Why is deprecated in the same enum as changeable?  It seems to me like we're talking about two completely independent things here.  Is the setting changeable at runtime and is the setting deprecated.

Reply:
Yes.

😄 

We'll address these in the config system refactor, just shoehorning in the user-facing functionality we need in the least invasive way for now.